File: SpringBoot-Shopping/shopping-manager-ms/src/main/java/com/gyf/shopping/service/BrandService.java
package com.gyf.shopping.service;

import com.gyf.shopping.pojo.Brand;

import java.util.List;
import java.util.Map;

public interface BrandService {

    public List<Brand> queryAllBrand();

    public Brand queryBrandById(Long id);

    public void createBrand(Brand brand);

    public void updateBrand(Brand brand);

    public void deleteManyBrand(Long[] ids);

    public List<Map> selectOptionList();



}

File: 前台页面/shopping-manager-web/service/brandService.js
app.service('bankService', function($http) {

	var manager_url = 'http://localhost:8899/manager-ms/';

	this.queryAllBrand = function() {

		return $http.get(manager_url + 'queryAllBrand');

	}

	this.deleteManyBrand = function(selectIds) {
		return $http.get(manager_url + "deleteManyBrand?ids=" + selectIds)
	};

	this.queryBrandById = function(brand_id) {

		return $http.get(manager_url + "queryBrandById/" + brand_id)
	}

	this.findByPage = function(pageNum, pageSize) {
		return $http.get(manager_url + 'findByPage?pageNum=' + pageNum + '&pageSize=' + pageSize)
	}

	this.createBrand = function(entity) {
		return $http.post(manager_url + "createBrand", entity)
	}
	
	this.updateBrand = function(entity) {
		return $http.post(manager_url + "updateBrand", entity)
	}

})